blockchain.go

```diff
@@ -445,6 +450,8 @@ func configureFVM(blockchain *Blockchain, conf config, blocks *blocks) (*fvm.Vir
 		fvmOptions...,
 	)
 
+	blockchain.coverageReport = ctx.ReusableCadenceRuntimePool.CoverageReport
+
```

Comment:
`./blockchain.go:453:61: ctx.ReusableCadenceRuntimePool.CoverageReport undefined (type "github.com/onflow/flow-go/fvm/runtime".ReusableCadenceRuntimePool has no field or method CoverageReport)`

Reply:
I will have to bump the `flow-go` version, with a release that includes the changes from here: https://github.com/onflow/flow-go/pull/4038